app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split(" ")) == 3:
```

Comment:
There is no need to specify that `split` should be performed over space `" "`, it is a default argument for `split`. Remove `" "`.